starter-core/src/main/java/io/micronaut/starter/feature/function/gcp/template/gcpFunctionGroovyJunit.rocker.raw

```diff
@@ -16,10 +16,20 @@ import io.micronaut.gcp.function.http.*
 class @project.getClassName()FunctionTest {
 
     @@Test
-    void testFunction() {
+    void testGet() {
         new HttpFunction().withCloseable { HttpFunction function ->
-           GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/@project.getPropertyName()")
-            assert response.status == HttpStatus.OK
+           GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/")
+            assertEquals(HttpStatus.OK, response.status)
+        }
+    }
+    @@Test
+    public void testPost(){
```

Comment:
Groovy default visibility modifier is public. Thus, you don't need to specify it.

Reply:
I've removed public